itest/starboard/starboard_cli_test.go

```diff
@@ -1102,6 +1121,7 @@ var _ = Describe("Starboard CLI", func() {
 			"cleanup",
 			"-v", starboardCLILogLevel,
 		}, GinkgoWriter, GinkgoWriter)
+		time.Sleep(4 * time.Second)
```

Comment:
We should use [Gomega Asynchronous Assertions](https://onsi.github.io/gomega/#making-asynchronous-assertions) instead of calling time.Sleep() explicitly

Reply:
I couldn't use with success the Eventually or Consistently. For testing purpose I'm still using the sleep. I will remove it